internal/template/template_utils_test.go

```diff
@@ -286,3 +286,34 @@ func Test_isListDefinition(t *testing.T) {
 		})
 	}
 }
+
+func TestEscapeJinjaTemplates(t *testing.T) {
```

Comment:
Please use table tests here. The test cases are more apparent, making the code easier to understand. 
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Reply:
In this case, table tests would not bring benefits.

Since this is more of a style issue and it deals with the test, I'll just leave it as it is.